scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -1201,7 +1201,7 @@
                sort-jobs-duration
                (->> tasks
                     (group-by util/task-ent->user)
-                    (pc/map-vals (fn [task-ents] (into (sorted-set-by task-comparator) task-ents)))
+                    (pc/map-vals (fn [task-ents] (sort task-comparator task-ents)))
```

Comment:
It should be impossible to have duplicates since the task ents are coming from the db where they are unique or generated for jobs where there is a one to one mapping from job to task and the job is unique in the database.